docs/simpleinjector/metricsCapturing/applyingMetricsCapturing.md

```diff
@@ -0,0 +1,65 @@
+# Applying Metrics-Capturing Concerns to Specific CQS Handler Implementations
+
+The metrics-capturing decorators for individual CQS handler types require one of the following as dependencies:
+- for query handlers, a corresponding [`IMetricsCapturingStrategyForQuery<TQuery, TResult>`](../../../src/Functional.CQS.AOP.MetricsCapturing/IMetricsCapturingStrategyForQuery.cs) implementation defined
+- for command handlers, a corresponding [`IMetricsCapturingStrategyForCommand<TCommand, TError>`](../../../src/Functional.CQS.AOP.MetricsCapturing/IMetricsCapturingStrategyForCommand.cs) implementation defined
+
+The above three contracts are defined by the `Functional.CQS.AOP.MetricsCapturing` NuGet package.  **As best practice, metrics-capturing implementations should be defined near the composition root**.  In other words, metrics-capturing strategy implementations should be defined in the same assembly containing the `Functional.CQS.AOP` installer code, or in a dedicated component registration assembly.  They should _not_ be defined in the same assembly as CQS handler parameter objects (`TQuery`, `TCommand`) or CQS handler implementations.  This is because metrics-capturing is an infrastructural concern, and thus belongs [at the boundary of the system and not within the application domain](http://jeffreypalermo.com/blog/the-onion-architecture-part-1/).
```

Comment:
"The above contracts", instead of "The above three contracts", since there's only 2 listed?

Reply:
Yep!  Fixed.